.github/ISSUE_TEMPLATE/issue-default.yml

```diff
@@ -1,34 +1,34 @@
 name: Issue
-description: Capture uncategorized work or content
+description: Describe an idea, feature, content, or non-bug finding
 
 body:
   - type: markdown
-    id: help
+    id: title-help
     attributes:
       value: |
-        > **Note**
-        > GitHub Issues use [GitHub Flavored Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for formatting.
+        > Titles should be short, descriptive, and compelling.
   - type: textarea
-    id: issue
+    id: issue-description
     attributes:
-      label: Issue Description
+      label: Issue description and context
       description: |
-        Describe the issue you are adding or content you are suggesting.
-        Share any next steps that should be taken our outcomes that would be beneficial.
+        Describe the issue so that someone who wasn't present for its discovery can understand the problem and why it matters. Use full sentences, plain language, and good [formatting](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax). Share desired outcomes or potential next steps. Images or links to other content/context (like documents or Slack discussion) are welcome.
     validations:
       required: true
   - type: textarea
-    id: additional-context
+    id: acceptance-criteria
     attributes:
-      label: Additional Context (optional)
-      description: "Include additional references (screenshots, design links, documentation, etc.) that are relevant"
+      label: Acceptance criteria
+      description: "If known, share 1-3 statements that would need to be true for this issue to be considered resolved. Use a [task list](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists#creating-task-lists) if appropriate."
   - type: textarea
-    id: issue-links
+    id: links-to-other-issues
     attributes:
-      label: Issue Links
+      label: Links to other issues
       description: |
-        What other issues does this story relate to and how?
-        
-        Example:
-        - 🚧 Blocked by: #123
-        - 🔄 Relates to: #234
\ No newline at end of file
+        Add the issue number of other issues this relates to and how (blocks, is blocked by, relates to).
```

Comment:
Open to feedback but I think the example here is helpful, especially because it standardizes emoji use - which sounds like a small thing but it really helps understand context quickly

Reply:
Re-added emoji in description text, and added emoji to the placeholder value